docs/design/TECHNICAL_DOCUMENTATION.md

```diff
@@ -0,0 +1,142 @@
+# Design tools technical documentation
+
+Otter framework exposes a set of tools to optimize the interactions between the designers and the developers.
+The purpose of these tools is to reduce the steps between the design of the solution and the integration to the applications.
+
+## Design Token
+
+The [Design Token specification](https://design-tokens.github.io/community-group/format/) define a dialog format between the design tools (such as [Figma](https://www.figma.com)) and a style language (CSS , Sass, Less, etc...).
+The Otter Framework provides **Builder**, **Schematic** and **Command Line Interface** to generate CSS Variable and/or Sass Variable, based on the inputted Design Token files.
+
+### Global architecture of the Parser and Renderer
+
+The Design Token code generator tool is technically split in two main features :
```

Comment:
Not sure why but I want to fix it with:
```suggestion
The Design Token code generator tool is technically split into two main features :
```
@sdo-1A: What do you think? :)

Reply:
I agree and remove the space between "features" and ":" --> `features:`

Reply:
I edited the suggestion